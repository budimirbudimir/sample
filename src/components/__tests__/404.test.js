import React from 'react'
import PageNotFound from '../404'
import renderer from 'react-test-renderer'
import { BrowserRouter } from 'react-router-dom'

it('Renders without crashing', () => {
	const tree = renderer
		.create(
			<BrowserRouter>
				<PageNotFound />
			</BrowserRouter>,
		)
		.toJSON()
	expect(tree).toMatchSnapshot()
})
