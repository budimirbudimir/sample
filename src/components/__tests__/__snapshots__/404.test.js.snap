// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without crashing 1`] = `
<div
  className="PageNotFound"
>
  <h1
    style={
      Object {
        "color": "#f4425c",
      }
    }
  >
    Error 404
  </h1>
  <h3>
    Requested page could not be found!
  </h3>
  <a
    href="/"
    onClick={[Function]}
    style={
      Object {
        "color": "#f4425c",
        "fontWeight": "700",
        "textDecoration": "none",
      }
    }
  >
    Go back to Home page
  </a>
</div>
`;
